src/invidious/user/imports.cr

```diff
@@ -228,8 +228,13 @@ struct Invidious::User
       subs = matches.map(&.["channel_id"])
 
       if subs.empty?
-        data = JSON.parse(body)["subscriptions"]
-        subs = data.as_a.map(&.["id"].as_s)
+        profiles = body.split('\n')
+        profiles.pop
```

Comment:
```suggestion
        profiles = body.split('\n', remove_empty: true
```

By doing so, so can be sure that no "profile" will is an empty string, and you ensure that if there is a single profile, it won't be dropped.